inference-engine/thirdparty/movidius/XLink/pc/protocols/usb_boot.c

```diff
@@ -657,11 +634,12 @@ static int send_file(libusb_device_handle *h, uint8_t endpoint, const uint8_t *t
         twb += wbr;
         p += wbr;
     }
-    if(usb_loglevel > 1)
-    {
-        double MBpS = ((double)filesize / 1048576.) / (elapsedTime * 0.001);
-        fprintf(stderr, "Successfully sent %u bytes of data in %lf ms (%lf MB/s)\n", filesize, elapsedTime, MBpS);
-    }
+
+#ifndef NDEBUG
+    double MBpS = ((double)filesize / 1048576.) / (elapsedTime * 0.001);
+    mvLog(MVLOG_DEBUG, "Successfully sent %u bytes of data in %lf ms (%lf MB/s)", filesize, elapsedTime, MBpS);
+#endif
```

Comment:
There you made logging conditional and only for `Debug` mode

Reply:
I want to avoid calculating of `MBpS` in release mode and please note the previous code:
```
if(usb_loglevel > 1)
{
....
}
```
where `usb_loglevel` always [zero](https://github.com/openvinotoolkit/openvino/pull/552/files/f62ded86b0e4c3c6451a21cce6e754d6769149e9#diff-9cb3a95be441ec664312f5bdfd1f3c82L73)